codegen/src/json_schema.rs

```diff
@@ -42,6 +43,7 @@ pub struct JsonObjectFieldInfo {
 #[derive(Clone, Debug)]
 pub struct JsonEnum {
     pub name: String,
+    pub untagged: bool,
```

Comment:
This is a mapping of the json schema format, so the `untagged` field would break from the specification.

Would it be enough to say if the variants have differing json schema types, they're untagged? In this case, "array" and "object". It should be good enough for now, and I'd prefer to not break the JSON schema er...schema. :)